tool-scripts/XProtectVersionCheck-EA.sh

```diff
@@ -16,7 +16,7 @@ online_json_url="https://sofafeed.macadmins.io/v1/macos_data_feed.json"
 user_agent="SOFA-Jamf-EA-XProtectVersionCheck/1.1"
 
 # local store
-json_cache_dir="/private/tmp/sofa"
+json_cache_dir="/var/tmp/sofa"
```

Comment:
can we get the non-symlink version of this path please?
```suggestion
json_cache_dir="/private/var/tmp/sofa"
```

```% ls -l /
total 10
<snip>
lrwxr-xr-x@  1 root  wheel    11 Sep  6 05:54 var -> private/var
```

Reply:
too easy 🙂

Reply:
Thanks! I don't actually have merge rights but I approve this change!